scripts/partition_manager/partition_manager.rst

```diff
@@ -96,34 +99,83 @@ placement: dict
          Place the partition after the first existing partition in the list.
 
      Valid values in the lists are ``app``, ``start``, ``end``, or the name of any partition.
+     The value ``start`` refers to the flash device's start address.
```

Comment:
The start address of the device's flash memory?

Reply:
Hmm, I'm using "flash" an adjective modifying "device", so how about:

```
[...] refers to the start address of the flash device's memory
```

Reply:
So we are talking about a "flash device" (like a USB stick)?

Reply:
We are talking about a flash device, yes. The start address here is an attribute of the flash device from the perspective of the nRF SoC's CPU. In particular, it's the first address in the CPU's memory map which refers to data stored on the flash device.

Reply:
ok